core/src/main/java/com/orange/ouds/core/theme/OudsTypography.kt

```diff
@@ -24,7 +24,9 @@ import com.orange.ouds.theme.currentWindowWidth
 import com.orange.ouds.theme.tokens.OudsTypographyKeyToken
 import com.orange.ouds.theme.tokens.semantic.OudsFontSemanticTokens
 
-
+/**
+ * @suppress
+ */
 data class OudsTypography(
```

Comment:
Instances of `OudsTypography` are exposed through `OudsTheme.typography`, so I think this should be documented.

I don't know if the constructor should be public though.

Same comment for other data classes of the same kind.